### PR TITLE
Aargs: serialize boolean

### PR DESCRIPTION
**Pull request by Thykof**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -5,7 +5,7 @@ root = true
 
 [*]
 indent_style = space
-indent_size = 4
+indent_size = 2
```
Comment (peterjah) [inline .editorconfig:8]: 🎉

File: assembly/std/arguments.ts [hunk 2]
```diff
@@ -46,7 +46,7 @@ export class Args {
   nextAddress(): Address {
     const length = this.nextU32();
     let address = Address.fromByteArray(
-      this.serialized.slice(this.offset as i32, (this.offset as i32) + length),
+      this.serialized.slice(this.offset, this.offset + length),
```
Comment (peterjah) [inline assembly/std/arguments.ts:49]: 🎉

**Review by peterjah — APPROVED** [review]

File: assembly/std/__tests__/arguments.spec.ts [hunk 3]
```diff
@@ -29,6 +29,16 @@ describe('Args tests', () => {
     expect(args2.nextU64()).toBe(97);
   });
```
Comment (SebastienValla) [inline assembly/std/__tests__/arguments.spec.ts:31]: is it a good idea to stay on aspect ?

Comment (Thykof) [reply assembly/std/__tests__/arguments.spec.ts:31]: not at all but we have an issue for migrating all aspect tests

**Review by SebastienValla — APPROVED** [review]

**Pull request merged**